src/utils/constants.ts

```diff
@@ -69,12 +72,30 @@ export const EMAIL_REGEX = new RegExp(
 );
 export const TTL_SECONDS_REGEX = new RegExp('^[0-9]{3,7}$');
 export const TTL_SECONDS_ENTRY_REGEX = new RegExp('^[0-9]{1,7}$');
+
+export const FQDN_REGEX = new RegExp(
+  '^(?:(?!-)[A-Za-z0-9-]{1,63}(?<!-)\\.)+[A-Za-z]{1,63}$',
+);
 export const ARNS_REGISTRY_ADDRESS = new ArweaveTransactionID(
```

Comment:
odd here that ARNS_REGISTRY_ADDRESS defaults to testnet, whiel IO_PROCESS_ID defaults to devnet

Reply:
ARNS_REGISTRY_ADDRESS was a smartweave var, removed it